app/Rules/OverrideStartScreenRule.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace ALttP\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+
+class OverrideStartScreenRule implements Rule
+{
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function passes($attribute, $value)
+    {
+        if (is_array($value)) {
+            return count($value) === 5 && min($value) >= 0x00 && max($value) <= 0x1F;
+        } else if ($value === false) {
+            return true;
+        }
+        return false;
```

Comment:
```php
        if (is_array($value)) {
            return count($value) === 5 && min($value) >= 0x00 && max($value) <= 0x1F;
        }

        return $value !== false;
```

Reply:
This doesn't quite do what I want it to.  I'm going to instead just have it return false and fail if it isn't an array and meets the conditions.